src/vs/workbench/contrib/extensions/electron-browser/extensionTipsService.ts

```diff
@@ -1132,5 +1132,87 @@ export class ExtensionTipsService extends Disposable implements IExtensionTipsSe
 			.map(extensionId => (<IExtensionRecommendation>{ extensionId, sources: ['application'] })));
 	}
 
+	// called when Visualizer icon is clicked in the Query Results grid; "Never Show Again" is not an option
+	promptVisualizerExtensions(): void {
```

Comment:
A few things:
- Any edit to the src/vs tree needs a SQL Carbon Edit comment block or it'll be deleted on next merge. Please verify any cases where you have this and add the comments.
- In this specific case, Let's not put this logic in the VS code area. There's no reason this couldn't just go directly in your action instead of in this service. I'd just move to the file you have the main action in for now.

Reply:
No, it's all good.